conf.d/abbr_tips.fish

```diff
@@ -31,6 +31,36 @@ function __abbr_tips --on-event fish_postexec -d "Abbreviation reminder for the
                 set -e __ABBR_TIPS_KEYS[$abb]
                 set -e __ABBR_TIPS_VALUES[$abb]
         end
+    else if test "$command[1]" = "alias"
+        # Update abbreviations list when adding aliases
+        set -l alias_key
+        set -l alias_value
+
+        if string match -q '*=*' "$command[2]"
+            if test (count $command) = 2
+                set command_split (string split = $command[2])
+                set alias_key "a__$command_split[1]"
+                set alias_value $command_split[2]
+                set -a alias_value $command[3..-1]
+            end
+        else
+            set alias_key "a__$command[2]"
+            set alias_value $command[3..-1]
+        end
+
+        if set -l abb (contains -i -- "$command[3..-1]" $__ABBR_TIPS_KEYS)
+            set __ABBR_TIPS_KEYS[$abb] $alias_key
+            set __ABBR_TIPS_VALUES[$abb] (string trim -c '\'"' -- $alias_value | string join ' ')
+        else
+            set -a __ABBR_TIPS_KEYS $alias_key
+            set -a __ABBR_TIPS_VALUES (string trim -c '\'"' -- $alias_value | string join ' ')
+        end
+    else if test "$command[1]" = "functions"
+        # Update abbreviations list when removing aliases
+        and string match -q -r '^--erase|-e$' -- "$command[2]"
+        and set -l abb (contains -i -- "$command[3]" $__ABBR_TIPS_KEYS)
```

Comment:
This won't erase the alias since we prefix them with `a__`. Something like `(contains -i  a__{$command[3]} -- $__ABBR_TIPS_KEYS)` should work.